worlds/earthbound/Options.py

```diff
@@ -0,0 +1,625 @@
+       Hunt: Items required to finish the Monkey Caves will needsell you every minor item needed to be found outside. They can be obtained from the Dusty Dunes drugstore, the Fourside department store, and the pizza shop in either Twoson or Threed.
+       Shop: The monkey outside the Monkey Caves will sell every item needed to complete the caves and is not affected by shop randomization.
+       Solved: The Monkey Caves monkeys will already be moved out of the way and not require any items."""
+    display_name = "Monkey Caves Mode"
+    option_chests = 0
+    option_hunt = 1
+    option_shop = 2
+    option_solved = 3
+    default = 1
+    
+
+class ShortenPrayers(DefaultOnToggle):
+    """If enabled, the Prayer cutscenes while fighting Giygas will be skipped, excluding the final one."""
+    display_name = "Skip Prayer Sequences"
+
+
+class RandomStartLocation(Toggle):
+    """If disabled, you will always start at Ness's house with no teleports unlocked.
+       If enabled, you will start at a random teleport destination with one teleport unlocked.
+       Additionally, you will need to fight Captain Strong to access the north part of Onett if this is enabled."""
+    display_name = "Random Starting Location"
+
+
+class LocalTeleports(Toggle):
+    """Forces all teleports and Poo PSI to be placed locally in your world."""
+    display_name = "Local Teleports"
+
+
+class CharacterShuffle(Choice):
+    """Shuffled: Characters will be shuffled amongst Character Locations. Extra locations will have Flying Man, a Teddy Bear, or a Super Plush Bear.
+       Anywhere: Characters can be found anywhere in the multiworld, and character locations will have regular checks.
+       See the Game Page for more information on Character Locations."""
+    display_name = "Character Shuffle"
+    option_shuffled = 0
+    option_anywhere = 1
+    default = 0
+
+
+class PSIShuffle(Choice):
+    """None: Characters will learn their normal PSI skills.
+       Basic: Offensive and Assist PSI will be shuffled. Recovery PSI is not modified. Ness's Favorite Thing will be named Wave in other slots.
+       Extended: Basic shuffle, but includes Jeff gadgets and some combat items.
+       See the Game Page for more information."""
+    display_name = "PSI Shuffle"
+    option_none = 0
+    option_basic = 1
+    option_extended = 2
+
+
+class BossShuffle(PlandoBosses):
+    """Shuffles boss encounters amongst each other."""
+    display_name = "Boss Shuffle"
+
+    option_false = 0
+    option_true = 1
+    default = 0
+    bosses = boss_plando_keys
+    locations = boss_plando_keys
+    duplicate_bosses = False
+
+    @classmethod
+    def can_place_boss(cls, boss: str, location: str) -> bool:
+        return True
```

Comment:
I'm a little bit confused about this. It's probably not hurting anything, but it probably isn't supposed to be here anymore?

Reply:
Oh ok, my bad!